### PR TITLE
fix(justfile): Delete app data with just wipe-app on new XCode 

### PR DESCRIPTION
**Pull request by klochowicz**
Upgrade in XCode changed the application directory.

### CODE REVIEW

File: coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql
Comment (bonomat) [inline coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql:3]: This could be a reference to `users`.`pubkey`

File: coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql
Comment (bonomat) [inline coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql:4]: Is the second `not null` redundant? 
```suggestion
    timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
```

File: mobile/lib/features/wallet/create_invoice_screen.dart
Comment (bonomat) [inline mobile/lib/features/wallet/create_invoice_screen.dart:245]: This dialog needs a bit more love, but this space here adds a space at the beginning of the next line.

```suggestion
                            "Please try again later.\nIf you have further questions, please contact us on Telegram: ",
```

File: coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql
Comment (bonomat) [inline coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql:6]: Once #1384 is in this should be a reference to the `id` of said liquidity option.

**Review by bonomat — COMMENTED** [review]
I think this goes into the right direction. 
Please either add the proposed improvements into this PR or right away in a follow-up PR. 

@holzeis : shall we wait for #1384 before this gets merged?

File: mobile/lib/features/wallet/create_invoice_screen.dart
Comment (holzeis) [inline mobile/lib/features/wallet/create_invoice_screen.dart:241]: 🎨 The default text style looks very odd. Please change to something that is more consistent to our otherwise used styles.

File: coordinator/src/routes.rs
Comment (holzeis) [inline coordinator/src/routes.rs:191]: 🔧 We should only insert a liquidity request and check the balance if the user intends to open a channel (onboarding). This will be introduced with https://github.com/get10101/10101/pull/1384. Please adapt once this has landed.

**Review by holzeis — COMMENTED** [review]
Not sure what we will use the liquidity requests entry for. But I guess we should only store failed ones? Also there seems to be a strong overlap with announced channels.

**Comment by holzeis** [comment]
> I think this goes into the right direction. Please either add the proposed improvements into this PR or right away in a follow-up PR.
> 
> @holzeis : shall we wait for #1384 before this gets merged?

yes, lets wait for #1384 to land.

File: mobile/lib/features/wallet/application/wallet_service.dart
Comment (klochowicz) [reply mobile/lib/features/wallet/application/wallet_service.dart:31]: that's not easy to pull off in FRB and not essential. I have left a TODO that eventually it would be nice to do it, but it works just as good, especially that there's a note to not change the comment in the other place.

File: coordinator/src/routes.rs
Comment (klochowicz) [reply coordinator/src/routes.rs]: I recommend looking at the whole PR.

File: coordinator/src/routes.rs
Comment (klochowicz) [reply coordinator/src/routes.rs:191]: In hindsight, this ticket should have only be started after #1384 got merged. Rebase was pretty painful :/

File: coordinator/src/routes.rs
Comment (klochowicz) [reply coordinator/src/routes.rs:189]: yes, that is how I understood the requirement. Are you OK with this?

File: coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql
Comment (klochowicz) [reply coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql]: great idea. Should we do similar with `trades.trader_pubkey` ?

**Comment by klochowicz** [comment]
The only thing left is the custom error; it can be done, but I don't think it's essential as I left a comment at the definition to not change that.

File: mobile/lib/features/wallet/application/wallet_service.dart
Comment (holzeis) [reply mobile/lib/features/wallet/application/wallet_service.dart:26]: FfiExceptions do have an error code. It's better to use the error code instead of matching the error message.

https://pub.dev/documentation/flutter_rust_bridge/latest/flutter_rust_bridge/FfiException-class.html

**Comment by holzeis** [comment]
@klochowicz sorry it got a little messy. I've extracted the first commit to another PR and also had to fix quite some things as it wasn't compiling. I left this PR for fixing the justfile.

File: justfile
Comment (bonomat) [inline justfile:188]: I assume this should have been 
```
    if [ -d "~/Library/Containers/" ]; then
```

**Comment by klochowicz** [comment]
> @klochowicz sorry it got a little messy. I've extracted the first commit to another PR and also had to fix quite some things as it wasn't compiling. I left this PR for fixing the justfile.

thanks for carrying it across the finish line!

**Review by bonomat — APPROVED** [review]

File: justfile
Comment (Restioson) [inline justfile:201]: Changing this to
```
    # If no path was found, use a dummy path to avoid errors
    if [[ -z ${MACOS_PATH+x} || ! $MACOS_PATH || ! ${MACOS_PATH//[[:space:]]/} ]]; then
```
Seems to work

Comment (luckysori) [reply justfile:201]: Nice find! Can you open a PR for this?

Comment (Restioson) [reply justfile:201]: #1465

**Pull request merged**
